### PR TITLE
Added kuttl test github actions

### PR DESCRIPTION
**Pull request by nsathyaseelan**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        with:
+          path: kyverno-policies
+
+      - name: Clone Enterprise Kyverno
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+        with:
+          repository: nirmata/kyverno
+          path: kyverno
+          # The target branch of a pull request or the branch/tag of a push
+          ref: ${{ github.base_ref || github.ref_name }}
```
Comment (anusha94) [inline .github/workflows/ci.yaml:26]: The PR will be in `kyverno-policies` repo and this line is about checking out the branch in `nirmata/kyverno` repo. I don't get why is this needed. This should simply checkout the release branch in kyverno that we want to test

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+      - name: Set up Go 
+        uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568
+        with:
+          go-version: 1.18
+
+      - name: Run Kyverno CLI tests
+        run: go run ./cmd/cli/kubectl-kyverno test ../kyverno-policies
```
Comment (anusha94) [inline .github/workflows/ci.yaml:34]: create a make target for this

File: best-practices/disallow_cri_sock_mount/resources/resource-1.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  containers:
+  - name: myshell
+    image: "ubuntu:18.04"
+    command:
+    - /bin/sleep
+    - "300"
+  volumes:
+  - name: dockersock
+    hostPath:
+      path: /var/run/docker.sock
```
Comment (anusha94) [inline best-practices/disallow_cri_sock_mount/resources/resource-1.yaml:15]: needs newline

File: best-practices/disallow_cri_sock_mount/resources/resource-1.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
```
Comment (pns-nirmata) [inline best-practices/disallow_cri_sock_mount/resources/resource-1.yaml:16]: Not sure why it shows this. Perhaps a tab or space on line 16?

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+.PHONY: get-kyverno-binary
+get-kyverno-binary: 
+	@echo Download kyverno binary ... >&2
+	@curl -LO https://github.com/nirmata/kyverno/releases/download/$(N4K_BINARY_VERSION)/kyverno-cli_$(N4K_BINARY_VERSION)_linux_x86_64.tar.gz
+	@tar -xvf kyverno-cli_$(N4K_BINARY_VERSION)_linux_x86_64.tar.gz
+
+.PHONY: run-cli-test
+run-cli-test: 
+	@echo wait kyverno pod status installation... >&2
+	@./kyverno test ../kyverno-policies
```
Comment (anusha94) [inline Makefile:85]: why `../kyverno-policies`? Can you simply do `.` -> for the current directory?

Comment (nsathyaseelan) [reply Makefile]: @anusha94  addressed the review comment

**Comment by anusha94** [comment]
LGTM.
